packages/js/src/browser/integrations/breadcrumbs.ts

```diff
@@ -247,15 +247,11 @@ export default function (_window = globalThisOrWindow()): Types.Plugin {
           })
         }
 
-        // https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onpopstate
-        instrument(_window, 'onpopstate', function (original) {
-          return function () {
+        if (typeof addEventListener === 'function') {
+          addEventListener('popstate', (_event) => {
```

Comment:
I think this is a better approach. `addEventListener` does not replace the function.